daal4py/sklearn/cluster/distributed_k_means_fit.py

```diff
@@ -0,0 +1,102 @@
+#
+#*******************************************************************************
+# Copyright 2014-2020 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#******************************************************************************/
+
+import daal4py
+from daal4py.engines.row_partition_actor import RowPartitionsActor
+from daal4py.engines.ray.ray_context import RayContext
+from daal4py.sklearn.utils.pyoneccl import PyOneCCL
+
+from modin.developer import unwrap_row_partitions
+
+import numpy as np
+import ray
+import modin.pandas as pd
+
+
+@ray.remote(num_cpus=1)
```

Comment:
What about modin on dask?

Reply:
Unwrapping partitions of a `DataFrame` works for dask engine as well. Some changes are needed to do here. It looks like it is feasible. Is it planned to be as part of this PR or another?